location_scrapbook/app/src/main/java/erica/locationscrapbook/DBService.java

```diff
@@ -36,31 +36,23 @@ public void addLoc(Marker marker) {
     public void deleteLoc (Marker marker) {
         SQLiteDatabase sql = db.getWritableDatabase();
 
-        StringBuilder sb = new StringBuilder();
-        sb.append("delete from ");
-        sb.append(LocationDbSchema.TABLE_NAME);
-        sb.append(" where ");
-        sb.append(LocationDbSchema.LAT_TITLE);
-        sb.append("=");
-        sb.append(marker.getPosition().latitude);
-        sb.append(" and ");
-        sb.append(LocationDbSchema.LON_TITLE);
-        sb.append("=");
-        sb.append(marker.getPosition().longitude);
+        String sb = "delete from " +
+                LocationDbSchema.TABLE_NAME +
+                " where " +
+                LocationDbSchema.LAT_TITLE +
+                "=" +
+                marker.getPosition().latitude +
+                " and " +
+                LocationDbSchema.LON_TITLE +
+                "=" +
+                marker.getPosition().longitude;
```

Comment:
Making this just a regular string instead of a StringBuilder object makes your app a little more efficient